### ISSUE TITLE
PlayFabMultiplayerWin.lib cannot read at 0x82

### ISSUE THREAD

**Issue by CHADALAK1** [PlayFab/PlayFabMultiplayerUnreal#14]
For some reason I cannot get UE 4.27 to read the PlayfabMultiplayerWin.lib file for the life of me on Win64 (Engine source and Binary versions of UE). It always comes back with the same error. 

`PlayFabMultiplayerWin.lib: [LNK1107] invalid or corrupt file: cannot read at 0x82`

Reproduceable on an empty Unreal project as well. Same error. (Tried versions 2.2.4, 2.2.1, and 2.2.0)
  
Wanted to raise this as an issue in case something happened to the .lib file for the Playfab OSS

Steps to reproduce (Engine Source):
-Following the steps [Here](https://learn.microsoft.com/en-us/gaming/playfab/features/multiplayer/networking/party-unreal-engine-oss-quickstart)
-Create a Empty Unreal project
-Enable the Playfab OSS via .uproject edit
-Build project

**Comment by nassosterz-ms** [comment]
Hello,

We are sorry you are facing this issue. Could you attach the log file of the build also?

**Comment by CHADALAK1** [comment]
[Log.txt](https://github.com/PlayFab/PlayFabMultiplayerUnreal/files/9694621/Log.txt)


Hello! Thank you for the response!! I've attached the log file for the build as well

**Comment by CHADALAK1** [comment]
I got the PlayFab OSS to work! 

First off, I wasn't aware I needed to use Git LFS to pull the plug-in from here. I selected Code->Download Zip via the browser. Now I know this it was much better.

HOWEVER, I noticed this only works via Engine Source and not Binary versions of UE4. Typically I add the plug-in into the project folder itself, but when opening the project if I do It tells me the OnlineSubsystemPlayFab plug-in isn't modified or compatible with the project. Keep in mind, this is the Binary version

If I use the Engine Source version of UE 4.27 , It works totally fine. Compiles and all is well!
